src/parser/priest/discipline/modules/spells/ShadowCovenant/ShadowCovenantOutput.tsx

```diff
@@ -0,0 +1,105 @@
+import SPELLS from 'common/SPELLS';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
+import calculateEffectiveHealing from 'parser/core/calculateEffectiveHealing';
+import Events, { DamageEvent, HealEvent } from 'parser/core/Events';
+import { Options } from 'parser/core/EventSubscriber';
+import AtonementDamageSource from '../../features/AtonementDamageSource';
+
+const { SHADOW_COVENANT_BUFF, SHADOW_COVENANT } = SPELLS;
+
+type ShadowCovenantPasslist = Set<number>;
+
+const SHADOW_COVENANT_DAMAGE_PASSLIST: ShadowCovenantPasslist = new Set([
+  SPELLS.SHADOW_WORD_PAIN.id,
+  SPELLS.SHADOW_WORD_DEATH.id,
+  SPELLS.SCHISM_TALENT.id,
+  SPELLS.MIND_SEAR.id,
+  SPELLS.MINDGAMES.id,
+  SPELLS.UNHOLY_TRANSFUSION.id,
+]);
+
+const SHADOW_COVENANT_HEALING_PASSLIST: ShadowCovenantPasslist = new Set([SPELLS.SHADOW_MEND.id]);
```

Comment:
I'm aware this set has one member but I think it's a sensible assumption to assume they'll start adding the new Covenant abilities to the buff list here

Reply:
Woudn't it be cleaner if we just added the spell school in our atonement generating abilities instead of having 2 lists ?